plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergMaterializedViewTest.java

```diff
@@ -401,9 +408,14 @@ public void testDetectStaleness()
     @Test
     public void testSqlFeatures()
     {
+        File tempDir = getDistributedQueryRunner().getCoordinator().getBaseDataDir().toFile();
         // Materialized views to test SQL features
-        assertUpdate("CREATE MATERIALIZED VIEW materialized_view_window WITH (partitioning = ARRAY['_date']) as select _date, " +
-                "sum(_bigint) OVER (partition by _date order by _date) as sum_ints from base_table1");
+        assertUpdate("CREATE MATERIALIZED VIEW materialized_view_window " +
+                "WITH (\n" +
+                "   partitioning = ARRAY['_date'],\n" +
+                "   location = '" + tempDir + "/iceberg_data/tpch/materialized_view_window_storage_table'\n" +
```

Comment:
Create the MV as it used to be created and update only the `SHOW CREATE ...`  assertion:

```java
Path schemaDirectory = getDistributedQueryRunner().getCoordinator().getBaseDataDir().resolve("iceberg_data/tpch");
        assertThat((String) computeScalar("show create materialized view  materialized_view_window"))
                .matches("\\QCREATE MATERIALIZED VIEW " + qualifiedMaterializedViewName + "\n" +
                        "WITH (\n" +
                        "   format = 'ORC',\n" +
                        "   format_version = 2,\n" +
                        "   location = '" + schemaDirectory + "/st_\\E[0-9a-f]+\\Q',\n" +
                        "   partitioning = ARRAY['_date']\n" +
                        ") AS\n" +
                        "SELECT\n" +
                        "  _date\n" +
                        ", sum(_bigint) OVER (PARTITION BY _date ORDER BY _date ASC) sum_ints\n" +
                        "FROM\n" +
                        "  base_table1");
```

Reply:
same in the other test